hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -160,10 +162,19 @@ private void handlePlatformTransaction(
 
             preHandleResult = getCurrentPreHandleResult(state, platformEvent, platformTxn, configuration);
             final var transactionInfo = preHandleResult.txInfo();
+            final var transaction = transactionInfo.transaction();
             final var txBody = transactionInfo.txBody();
+            final Bytes transactionBytes;
+
+            if (transaction.signedTransactionBytes().length() > 0) {
```

Comment:
Might be worth checking for null here, in case this transaction doesn't have the new signed bytes form.
```suggestion
            if (transaction.signedTransactionBytes() != null && transaction.signedTransactionBytes().length() > 0) {
```

Reply:
transaction is a PBJ record, the default value of signedTransactionBytes is Bytes.EMPTY. It will not be null unless someone set it to null in the Builder. I was thinking about we should also check both transactionInfo and transaction to see if they are null, then throw HandleException or something. I didn't it add yet and like to see reviewers' comments first. Thanks.